README.md

```diff
@@ -266,21 +265,18 @@ Enterprise](https://konghq.com/kong-enterprise-edition/).
 
 ## License
 
-```
```

Comment:
The code section should also not be removed.  We want LICENSE text be formatted as code, so that it is clearly to see what is LICENSE and what is not.